spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -763,7 +763,7 @@ bom {
 		links {
 			site(version -> "https://jakarta.ee/specifications/persistence/%s.%s"
 				.formatted(version.major(), version.minor()))
-			javadoc(version -> "https://jakarta.ee/specifications/persistence/%s.%s/apidocs"
+			javadoc(version -> "https://jakarta.ee/specifications/persistence/%s.%s/apidocs/jakarta.persistence"
```

Comment:
Instead of appending "jakarta.persistence" to the url, here you could leverage the third argument to .formatted(...) and use:

`"https://jakarta.ee/specifications/persistence/%s.%s/apidocs/%s".formatted(...)`

Reply:
Thanks for the comments, but this PR has already been merged. I also think the changes are better as they are now. Nothing's being appended and `jakarta.persistence` is an unchanging part of the string so there's no benefit in it being a format argument.